.travis.yml

```diff
@@ -1,27 +1,16 @@
 sudo: required
```

Comment:
Why do we need sudo?

Reply:
We don't need actually. I needed it in my previous configuration and forgot to delete.